### PR TITLE
HyperShift: control-plane upgrade 4.12 to 4.12

### PR DESCRIPTION
**Pull request by LiangquanLi930**
https://issues.redhat.com/browse/OCPQE-14533

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-4.12-upgrade-from-stable-4.12-aws-ipi-ovn-hypershift-control-plane-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-4.12-upgrade-from-stable-4.12-aws-ipi-ovn-hypershift-control-plane-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-4.12-upgrade-from-stable-4.12-aws-ipi-ovn-hypershift-control-plane-p1-f7

**Comment by LiangquanLi930** [comment]
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/37732/rehearse-37732-periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-4.12-upgrade-from-stable-4.12-aws-ipi-ovn-hypershift-control-plane-p1-f7/1640687996686569472

**Comment by LiangquanLi930** [comment]
@kuiwang02 Could you help to review this PR? thanks

**Comment by LiangquanLi930** [comment]
@liangxia, Could you help to review this PR? thanks

File: ci-operator/step-registry/cucushift/upgrade/hypershift/control-plane/toimage/cucushift-upgrade-hypershift-control-plane-toimage-commands.sh [hunk 1]
```diff
@@ -121,6 +120,8 @@ done
 
 if [ $_upgradeReady -ne 0 ]; then
     echo "HyperShift HostedCluster(CP) upgrade failed"
+    sleep 3600
+    sleep 3600
```
Comment (liangxia) [inline ci-operator/step-registry/cucushift/upgrade/hypershift/control-plane/toimage/cucushift-upgrade-hypershift-control-plane-toimage-commands.sh:124]: Sleep two hours in CI ??

**Comment by kuiwang02** [comment]
I have same comment with @liangxia . it seems debug code.

for others, iamok from golang's view.

**Comment by LiangquanLi930** [comment]
@liangxia @kuiwang02 Thanks for your comments, Could you help to review again? Thanks

**Pull request merged**